app/assets/stylesheets/cart.css.scss

```diff
@@ -30,11 +30,12 @@
 }
 
 #cart-table {
-    margin-left:50px;
-    width: 850px;
-    border: none;
-    border-radius: 5px;
-    box-shadow: 0px 10px 20px rgba(0,0,0,.3);
+  margin-left:50px;
+  width: 850px;
+  border: solid 1px #696969;
```

Comment:
Color literals like `#696969` should only be used in variable declarations; they should be referred to via variable everywhere else.